apps/condo/domains/meter/gql.js

```diff
@@ -34,6 +34,14 @@ const METER_READING_FILTERS_FIELDS = '{ organization address accountNumber place
 const METER_READING_FILTER_TEMPLATE_FIELDS = `{ name employee { id } fields ${METER_READING_FILTERS_FIELDS} ${COMMON_FIELDS} }`
 const MeterReadingFilterTemplate = generateGqlQueries('MeterReadingFilterTemplate', METER_READING_FILTER_TEMPLATE_FIELDS)
 
+// TODO(codegen): write return type result!
+
+const DELETE_METER_AND_METER_READINGS_MUTATION = gql`
+    mutation _internalDeleteMeterAndMeterReadings ($data: _internalDeleteMeterAndMeterReadingsInput!) {
+        result: _internalDeleteMeterAndMeterReadings(data: $data) ${ METER_READING_FIELDS }
```

Comment:
no need to use _internal and mutation name needs to start with a verb
deleteMeterAndMeterReadings 
maybe, deleteMetersAndMeterReadingsForProperties

Reply:
this is to differentiate that this mutation is only used internally (another example is mutation for synchronizing contacts -- https://github.com/open-condo-software/condo/pull/3401)